src/Autofac.Integration.Wcf/IPerInstanceContextJitModuleContainer.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Autofac.Core;
+using System.Collections.Generic;
+
+namespace Autofac.Integration.Wcf
+{
+    /// <summary>
+    /// A list of just in time (JIT) registrations. This allows for the
+    /// current OperationContext or WebOperationContext or any item that is
```

Comment:
Please use `<see cref="..." />` to hyperlink references to other classes/properties - `OperationContext`, `WebOperationContext`, `InstanceContext`.